src/modules/trade/pure/TradeNumberInput/index.tsx

```diff
@@ -12,10 +12,20 @@ export interface TradeNumberInputProps extends TradeWidgetFieldProps {
   min?: number
   max?: number
   placeholder?: string
+  prefixComponent?: React.ReactElement
 }
 
 export function TradeNumberInput(props: TradeNumberInputProps) {
-  const { value, suffix, onUserInput, placeholder = '0', decimalsPlaces = 0, min, max = 100_000 } = props
+  const {
+    value,
+    suffix,
+    onUserInput,
+    placeholder = '0',
+    decimalsPlaces = 0,
+    min,
+    max = 100_000,
```

Comment:
why max is 100K?

Reply:
https://github.com/cowprotocol/cowswap/pull/2685#issuecomment-1621810939

Reply:
a bit arbitrary, but ooook

Reply:
100% arbitrary yes!

Open to suggestions

Reply:
Is there any reason why we would ever want to have an order with more than 100k parts?